qa/vagrant-helpers.rb

```diff
@@ -46,15 +46,15 @@ def self.run!(cmd)
   class VagrantHelpers
 
     def self.halt(machines="")
-      CommandExecutor.run!("vagrant halt #{machines.join(' ')}")
+      CommandExecutor.run!("vagrant halt #{Array(machines).join(' ')}")
```

Comment:
not I wonder if this is something we want to do here or also improve the default param value. I guess both are going to be need.